### PR TITLE
bug(docker): Change Node Version

### PR DESCRIPTION
**Pull request by codydaig**
@zymplsi @ilan

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:0.10
+FROM node:0.12
 
 MAINTAINER Matthias Luebken, matthias@catalyst-zero.com
```
Comment (ilanbiala) [inline Dockerfile]: What's this part?

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:0.10
+FROM node:0.12
```
Comment (ilanbiala) [inline Dockerfile]: Will this only work with 0.12, or also anything <= 4?

File: Dockerfile (hunk 1 above)
Comment (codydaig) [inline Dockerfile]: No idea. Been there for a while. Can probably remove it.

File: Dockerfile (hunk 2 above)
Comment (codydaig) [inline Dockerfile]: I've tested this personally with 0.12. The docker image does support 4, however, I have not personally tested it yet.

Comment (ilanbiala) [inline Dockerfile]: Okay, so let's just bump to 0.12.

File: Dockerfile (hunk 1 above)
Comment (ilanbiala) [inline Dockerfile]: Yeah let's remove

**Comment by ilanbiala** [comment]
LGTM once those two comments are addressed.

**Comment by lirantal** [comment]
+1, 0.10 is considered quite old by now.

**Comment by codydaig** [comment]
@ilanbiala @lirantal Updated.

**Comment by mleanos** [comment]
LGTM

**Comment by ilanbiala** [comment]
LGTM.

**Pull request merged**